qp_target_gene/support_files/patches/QIIME_to_QIIMEq2.sql

```diff
@@ -0,0 +1,68 @@
+-- With the QIIMEq2 release, the parameters for the command "Pick closed-reference
+-- OTUs" changed. The parameter "reference" has ben substituted for 2 parameters
+-- "reference-tax" and "reference-seq" which contains the path to the actual
+-- reference taxonomy file and the reference sequence file
+
+DO $do$
+DECLARE
+    potu_cmd_id     BIGINT;
+    new_potu_cmd_id BIGINT;
+    a_info          RECORD;
+    parameters      JSON;
+    ref_seqs        VARCHAR;
+    ref_taxa        VARCHAR;
+    pj_id           UUID;
+    input_data      VARCHAR;
+BEGIN
+    SELECT command_id INTO potu_cmd_id
+        FROM qiita.software_command sc
+            JOIN qiita.software s USING (software_id)
+        WHERE s.name = 'QIIME' AND s.version = '1.9.1' AND sc.name = 'Pick closed-reference OTUs';
+
+    SELECT command_id INTO new_potu_cmd_id
+        FROM qiita.software_command sc
+            JOIN qiita.software s USING (software_id)
+        WHERE s.name = 'QIIMEq2' AND s.version = '1.9.1' AND sc.name = 'Pick closed-reference OTUs';
+
+    -- Modify those artifacts and jobs that have been picked against Greengenes
+    FOR a_info IN
+        SELECT * FROM qiita.artifact WHERE command_id = potu_cmd_id
+    LOOP
+        IF a_info.command_parameters->>'reference' = '1' THEN
+            -- Greengenes
+            ref_seqs := '/databases/gg/13_8/rep_set/97_otus.fasta';
+            ref_taxa := '/databases/gg/13_8/taxonomy/97_otu_taxonomy.txt';
+        ELSIF a_info.command_parameters->>'reference' = '2' THEN
+            -- Silva
+            ref_seqs := '/projects/qiita_data/reference/silva_119_Silva_119_rep_set97.fna';
+            ref_taxa := '/projects/qiita_test_data/reference/silva_119_taxonomy_97_7_levels.txt';
+        ELSE
```

Comment:
Should this be an ELSIF and print/warn about the ELSE?

Reply:
Good point - done